src/Hl7.Fhir.Serialization/FhirXmlNode.cs

```diff
@@ -438,7 +438,37 @@ object checkRepresentation(ITypedElement node, IExceptionSource ies, object _)
         private ExceptionNotification buildException(string message) => ExceptionNotification.Error(
                 new StructuralTypeException("Parser: " + message));
 
-        public string OriginalElement => ((XElement)Current).ToString(SaveOptions.DisableFormatting);
+        public string OriginalElement
+        {
+            get {
+                if (((XElement)Current.Parent).Name.Namespace != ((XElement)Current).Name.Namespace)
+                {
+                    return ((XElement)Current).ToString(SaveOptions.DisableFormatting);
+                }
+                else
+                {
+                    var test = StripNamespaces((XElement)Current);
+                    return (test.ToString(SaveOptions.DisableFormatting));
+                }
+            }
+            
+        }
+
+        private XElement StripNamespaces(XElement rootElement)
```

Comment:
I have no idea what and why this is, can we add documentation?

Reply:
It feels like this property is expensive with all the namespace manipulation. We should cache the result I think.

Reply:
I don't agree we should remove the namespace - but it's too late to change this now. Will mark all of this functionality as [Obsolete] so we can review this in the next version of the API.